app/src/main/kotlin/com/exallium/todoapp/notedetail/NoteDetailViewImpl.kt

```diff
@@ -0,0 +1,72 @@
+package com.exallium.todoapp.notedetail
+
+import android.os.Bundle
+import android.widget.TextView
+import butterknife.BindView
+import com.exallium.todoapp.R
+import com.exallium.todoapp.app.TodoApp
+import com.exallium.todoapp.entities.Note
+import com.exallium.todoapp.mvp.BaseViewImpl
+import timber.log.Timber
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * Conductor Controller displaying a single note
+ */
+class NoteDetailViewImpl(bundle: Bundle) : BaseViewImpl<NoteDetailView, NoteDetailPresenter, NoteDetailViewImpl, NoteDetailComponent>(bundle), NoteDetailView {
+    object Constants {
```

Comment:
Don't do this.  Use ScreenBundle to manage Bundle KV pairs.  Toolbar title should come from a resource ID, not from a constant value.